.changeset/calm-traces-rest.md

```
@@ -0,0 +1,5 @@
+---
+'mastra': patch
+---
+
+Fixed trace details requesting usage breakdowns when usage columns are hidden, and stabilized trace panel action tests.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)